### ISSUE TITLE
Another super-init-not-called false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#389]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

The following code emits a super-init-not-called.

```
#!python

import ctypes

class Foo(ctypes.BigEndianStructure):
    def __init__(self):
        ctypes.BigEndianStructure.__init__(self)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/389
